### PR TITLE
Chess board

### PR DESCRIPTION
**Pull request by ieric01**
Fixed the following:
-Border-Collapse applied from bootstrap.

### CODE REVIEW

**Comment by gabihuizar** [comment]
works on my end now! :smile:

File: app/controllers/games_controller.rb [hunk 1]
```diff
@@ -8,7 +8,8 @@ def create
 	end
 
 	def show
-		@game = Game.find(params[:id])
+		# @game = Game.find(params[:id])
+    @game ||= Game.where(id: params[:id]).last
```
Comment (mscoutermarsh) [inline app/controllers/games_controller.rb]: For this you can do

`@game = Game.find(params[:id])`

File: spec/models/piece_spec.rb [hunk 2]
```diff
@@ -94,4 +92,43 @@
       end
     end
   end
+
+  describe '#move_to!' do
+    let(:piece) { build(:piece) }
```
Comment (mscoutermarsh) [inline spec/models/piece_spec.rb]: you could remove some duplication here by changing this to

```
let(:black_piece) { build(:piece, status: 'black') }
let(:white_piece) { build(:piece, status: 'black') }
```

Then use them below :)

File: spec/models/piece_spec.rb [hunk 3]
```diff
@@ -94,4 +92,43 @@
...
+        expect{piece.move_to!('H', 4)}.to change{Piece.count}.from(2).to(1)
+        expect(piece).to have_attributes(x: 'H', y: 4)
+      end
+       it 'piece2 destroyed and piece has updated xy coordinates' do
+        piece.update(x: 'A', y: 2, status: 'white')
+        piece2.update(x: 'A', y: 6, status: 'black')
+
+        expect{piece.move_to!('A', 6)}.to change{Piece.count}.from(2).to(1)
+        expect(piece).to have_attributes(x: 'A', y: 6)
+      end
```
Comment (mscoutermarsh) [inline spec/models/piece_spec.rb]: the indenting here looks a bit off :)

File: spec/models/piece_spec.rb [hunk 4]
```diff
@@ -94,4 +92,43 @@
...
+        piece2.update(x: 'H', y: 4, status: 'white')
+
+        expect{piece.move_to!('H', 4)}.to change{Piece.count}.from(2).to(1)
+        expect(piece).to have_attributes(x: 'H', y: 4)
+      end
+       it 'piece2 destroyed and piece has updated xy coordinates' do
+        piece.update(x: 'A', y: 2, status: 'white')
+        piece2.update(x: 'A', y: 6, status: 'black')
+
+        expect{piece.move_to!('A', 6)}.to change{Piece.count}.from(2).to(1)
```
Comment (mscoutermarsh) [inline spec/models/piece_spec.rb]: nice :)

File: spec/models/piece_spec.rb (hunk 2 above)
Comment (mscoutermarsh) [inline spec/models/piece_spec.rb]: noticing this won't work for the last tests (our piece). Could create new `lets` for that context.

File: spec/models/piece_spec.rb [hunk 5]
```diff
@@ -94,4 +92,43 @@
...
+        expect(piece).to have_attributes(x: 'A', y: 6)
+      end
+    end
+
+    context 'our piece' do
```
Comment (mscoutermarsh) [inline spec/models/piece_spec.rb]: nice use of context btw :)

**Comment by mscoutermarsh** [comment]
left comments :smile:, good work. Merge it whenever you're ready

**Pull request merged**